Dockerfile

```diff
@@ -25,14 +25,15 @@ ENV PATH=${ANDROID_NDK}:${ANDROID_HOME}/cmdline-tools/tools/bin:${ANDROID_HOME}/
 # Install system dependencies
 RUN apt update -qq && apt install -qq -y --no-install-recommends \
         apt-transport-https \
-        build-essential \
         curl \
         file \
         git \
+        g++ \
```

Comment:
Looks like gcc is missed.

Reply:
g++ depends on gcc, but lets add in the packages